samples/algorithms/Grover.qs

```diff
@@ -24,8 +27,12 @@ operation Main() : Result[] {
     let iterations = CalculateOptimalIterations(nQubits);
```

Comment:
About the comment just above this: Is it worth mentioning what would happen if I set the number of iterations to a _higher_ value? Is that interesting?